pages/common/ippeveprinter.md

```diff
@@ -0,0 +1,29 @@
+# ippeveprinter
+
+> A simple IPP Everywhere printer server.
```

Comment:
```suggestion
> A simple IPP everywhere printer server.
```

Reply:
I think "Everywhere" should stay this way, IPP Everywhere™ is a trademark.

Reply:
Oh ok, thanks for the information. Will resolve this.